### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**
"enrollmentType" > "ownerType"
"Corporate" > "Company" (x2)

Should also tidy up the purple note box at the bottom of [this page](https://docs.microsoft.com/en-us/azure/active-directory/enterprise-users/groups-dynamic-membership#rules-for-devices) which is trying to cater towards these typos that are now fixed.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@JeremyTBradshaw : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/developer/reports-ref-devices.md [hunk 1]
```diff
@@ -163,13 +163,13 @@ The **EnrollmentFailureReason** entity indicates a more detailed reason for a de
 | APNSCertificateExpired           | Apple devices cannot be managed with an expired Apple MDM push certificate.                                                                                                                            |
 ## ownerTypes
 
-The **enrollmentType** entity indicates whether a device is corporate, personally owned, or unknown.
+The **onwerType** entity indicates whether a device is corporate, personally owned, or unknown.
```
Comment (Jak-MS) [inline memdocs/intune/developer/reports-ref-devices.md:166]:
```suggestion
The **ownerType** entity indicates whether a device is corporate, personally owned, or unknown.
```

**Comment by PRMerger14** [comment]
@Erikre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**